Essentials/src/messages.properties

```diff
@@ -379,7 +379,7 @@ onlyPlayerSkulls=\u00a74You can only set the owner of player skulls (\u00a7c397\
 onlySunStorm=\u00a74/weather only supports sun/storm.
 openingDisposal=\u00a76Opening disposal menu...
 orderBalances=\u00a76Ordering balances of\u00a7c {0} \u00a76users, please wait...
-oversizedMute=�4You may not mute a player for this period of time.
+oversizedMute=\u00a74You may not mute a player for this period of time.
```

Comment:
This isn't relevant to this PR. If there is an issue with this message, please open another PR.